plugins/reset/EraseEverything.qml

```diff
@@ -43,7 +43,7 @@ Component {
             id: action
             text: i18n.tr("Erase & Reset Everything")
             objectName: "factoryResetAction"
-            color: UbuntuColors.red
+            color: theme.palette.normal.negative
```

Comment:
This change, and its equivalent in ResetLauncherHome and ResetAllSettings, cause our new old friend to occur:

![Reset dialog, showing black text on red button and white text on gray button](https://user-images.githubusercontent.com/21966173/72740283-fb7bfb00-3b6a-11ea-9c89-3021779a6bac.png)

Reply:
Oops

Reply:
I checked again and looks ok in mine. Do you have latest ubuntu-toolkit changes, right?

Reply:
Completely reinstalled the system, same thing. Are you expecting another PR to be installed?

Reply:
Weird, the responsible for the cancel button color is the uitk. In theory if [this is installed](https://github.com/ubports/ubuntu-ui-toolkit/pull/64) it should look good.

This PR is previous to the uitk one, does that make any difference?

Reply:
Yes, that was indeed the issue. After the rebuild today, it works correctly. Strange dependencies, maybe.